components/camel-swagger-java/src/main/java/org/apache/camel/swagger/RestSwaggerReader.java

```diff
@@ -224,6 +224,9 @@ private void parse(Swagger swagger, RestDefinition rest, String camelContextId,
         // use annotation scanner to find models (annotated classes)
         for (String type : types) {
             Class<?> clazz = classResolver.resolveClass(type);
+            if (clazz == null) {
+                throw new RuntimeException("The ClassResolver could not resolve the model class of type: " + type);
```

Comment:
i would suggest RuntimeCamelException

Reply:
I think there is a resolveMandatoryClass method that should be used instead

Reply:
Agree resolveMandatoryClass provides what I'm looking for.  Will update.